lib/aws_extensions/kinesis/StreamDescription.rb

```diff
@@ -0,0 +1,12 @@
+module AwsExtensions
+  module Kinesis
+    module StreamDescription
+
+      # Public: Get the list of open shards sorted by hash key range, ascending
+      def sorted_shards
+        self.shards.select { |shard| shard.sequence_number_range.ending_sequence_number.nil? }
+                   .sort_by { |shard| shard.hash_key_range.starting_hash_key.to_i(10) }
```

Comment:
Yeah the default is 10. So it's actually kind of confusing because all the docs say these numbers are going to be md5 hashes so I originally had it as 16. But they already changed them to base 10, and then made them a string for some reason.